opengb/seed-core
# GRID-5167: :twisted_rightwards_arrows: Update to v2.17.4

#### Any background context you want to provide?
We (OPEN) maintain our own forked version of SEED with customizations for our clients. We want to combine v2.17.4 with our customizations.

#### What's this PR do?
This PR updates our customized version of SEED to version v2.17.4.

#### How should this be manually tested?
1. Run `docker build .` in the main directory of this Git repo.
2. Run `docker images`.
3. The new SEED Docker image will show at the top of the list, as the one with the most recent "CREATED" timestamp. Note the long, alphanumeric ID in the "IMAGE ID" column.
4. Run `docker tag {{ alphanumeric ID }} opengb/seed:{{ some unique version name }}`. You will need to decide what to use as the `some unique version name`. Usually the unique version name will be something like `grid-5167_opengb-develop_v2.17.4`, since that summarizes a lot of useful info.
5. Run `docker push opengb/seed:{{ some unique version name }}`.
6. In the main GRID Git repo, go to `systems/seed-docker`. 
7. Update the versions specified in the `services` -> `web` -> `image` and `services` -> `web-celery` -> `image` values of the `docker-compose.yml` file in that directory. Save the file.
8. Run `make server` in that directory to start running the new version of SEED.
9. Keep an eye on the logs, in case any errors come up while SEED is starting.
10. If there are no errors, go to `http://localhost:9000/` to reach SEED, then log in with `ops@opentech.eco`/`super-secret-password`.
11. Look around the interface in SEED, try uploading properties from Excel, and viewing data. Make sure nothing's broken.
12. If everything looks good, this PR can be merged in, and SEED Staging and Production can be updated to use the new Docker image. Deployment for those environments will be covered in another document.

#### What are the relevant tickets?
See [GRID-5167](https://opentech-cast.monday.com/boards/1256768982/pulses/4133255167)

#### Screenshots (if appropriate)
N/A
